es_ES/_data/catalog/widgets.json

```diff
@@ -0,0 +1,1582 @@
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/AnimatedContainer-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "AnimatedCrossFade",
+    "description": "Un widget que hace desvanecimientos cruzados entre dos hijos dados y realiza animaciones en si mismo entre sus tamaños.",
+    "categories": [
+      "Animation and Motion"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/AnimatedCrossFade-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Hero",
+    "description": "Un widget que marca a su hijo como candidato para las animaciones de Hero.",
+    "categories": [
+      "Animation and Motion"
+    ],
+    "subcategories": [
+      "Routing"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Hero-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "AnimatedBuilder",
+    "description": "Un widget de propósito general para construir animaciones. AnimatedBuilder es útil para widgets más complejos que deseen incluir una animación como parte de una función de construcción más grande. Para usar AnimatedBuilder, simplemente construya el widget y pásele una función de constructor.",
+    "categories": [
+      "Animation and Motion"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/AnimatedBuilder-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Draggable",
+    "description": "Un widget que se puede arrastrar desde un DragTarget. Cuando un widget arrastrable reconoce el inicio de un gesto de arrastre, muestra un widget de retroalimentación que rastrea el dedo del usuario a través de la pantalla. Si el usuario levanta el dedo mientras está en la parte superior de un DragTarget, se le da la oportunidad de aceptar los datos transportados por el objetivo arrastrable.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Draggable-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "LongPressDraggable",
+    "description": "Hace que su hijo sea arrastrable a partir de una pulsación larga.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/LongPressDraggable-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "GestureDetector",
+    "description": "Un widget que detecta gestos. Intenta reconocer gestos que corresponden a sus llamadas de retorno no nulas. Si este widget tiene un hijo, se remite a ese hijo por su comportamiento de tamaño. Si no tiene un hijo, crece para adaptarse a los padres.",
```

Comment:
A widget that detects gestures. Attempts to recognize gestures that correspond to its non-null callbacks. If this widget has a child, it defers to that child for its sizing behavior. If it does not have a child, it grows to fit the parent instead.
Un widget que detecta gestos. Intenta reconocer gestos que se corresponden con sus callbacks no nulos. Si este widget tiene un hijo, difiere a este hijo su comportamiento para dimensionarse. Sino tieen un hijo, crecerá hasta rellenar el widget padre.